.env.example

```
@@ -3,6 +3,7 @@ NEXT_PUBLIC_APP_NAME=MimeBookmark
 NEXT_PUBLIC_APP_URL=http://localhost:3000
 
 # Stripe (Required for paid features)
+STRIPE_PUBLISHABLE_KEY=pk_test_xxx
 NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_xxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY key should go before the STRIPE_PUBLISHABLE_KEY key

(UnorderedKey)